scripts/manifest.arm64.yml

```diff
@@ -0,0 +1,32 @@
+---
+version: 1.0.0.0-rc1
```

Comment:
1.0.0-rc1? If we're following semver then it should be three digits.

Reply:
The entire manifest will change per release, this is just a sample at this moment. Should I change it to 1.0.0?